src/main/java/com/jonahseguin/drink/provider/BooleanProvider.java

```diff
@@ -60,7 +61,7 @@ public String argumentDescription() {
     }
 
     @Override
-    public List<String> getSuggestions(@Nonnull String prefix) {
+    public List<String> getSuggestions(CommandSender sender, @Nonnull String prefix) {
```

Comment:
with the method overloading in DrinkProvider, this change in signature would be unnecessary